packages/fiori-elements-writer/test/__snapshots__/alp.test.ts.snap

```diff
@@ -5340,20 +5340,22 @@ xmlns:edmx=\\"http://docs.oasis-open.org/odata/ns/edmx\\">
   },
   \\"sap.ui.generic.app\\": {
     \\"pages\\": {
-      \\"ListReport|SEPMRA_C_ALP_SlsOrdItemCubeALPResults\\": {
+      \\"AnalyticalListPage|SEPMRA_C_ALP_SlsOrdItemCubeALPResults\\": {
```

Comment:
Where is this coming from? Is it just a change in the test resulting into this new page id? I guess it doesn't matter what the exact string is, as long as it is unique but still wondering if this is expected.

Reply:
Just read the issue, and understand it now. Thanks!

Reply:
The test was setting the wrong template type - thats part of the fix here.